app/client/liveChat.ts

```diff
@@ -0,0 +1,68 @@
+// <style type='text/css'>
+//     .embeddedServiceHelpButton .helpButton .uiButton {
+//         background-color: #005290;
+//         font-family: "Arial", sans-serif;
+//     }
+//     .embeddedServiceHelpButton .helpButton .uiButton:focus {
+//         outline: 1px solid #005290;
+//     }
+// </style>
+
+// tslint:disable:variable-name
```

Comment:
This is disabling the variable name rule, which `embedded_svc` does not conform to.

Reply:
Because it's not camel case?

Reply:
`embedded_svc` gets declared in the function signature so its code we control. You can change the name to something that conforms to our lint rules. No need to stick to the code snippet word-for-word./

Reply:
The variable name cannot be changed without breaking functionality, I have tried to do that. Likely due to being referenced by the script that is being added.

Reply:
Changing the variable name inside `initLiveChat` will indeed break functionality it needs to access `window.embedded_svc` but changing the name inside `initESW` will not because the scope of `embedded_svc` is that functional.

Reply:
Ah yes I'd changed it throughout in an effort to maintain consistency then went for the linting rule. You think it's better practice to change the `initESW` parameter and subsequent references to it within the function to camel case to avoid adding the linting rule?

Reply:
I do. That way we decrease the number of exceptions to our linting rule. We can also be more specific when we do disable the rule by using the next-line method or disabling/enabling the rule for a smaller piece of code.

Reply:
Right now there is one rule `tslint:disable-next-line:variable-name` which handles the linting for the snake case variable name. The global `tslint:disable:no-object-mutation` had already been changed to 5 instances of `tslint:disable-next-line:no-object-mutation` by this point.

Reply:
We can get rid of the `variable-name` rule exception by changing the name of the `embedded_svc` variable in the `initESW` function. This is code we control and this variable is in the function's scope so there's no need to introduce exceptions to our linting rules.

Reply:
I'm inclined to agree with @ripecosta here, renaming `embedded_svc: any` on line 12 to something like `liveChatAPI: any` better explains what's been passed into the function, and saves you having to do include the linter exclusion comment.